tasks/main.yml

```diff
@@ -24,6 +22,11 @@
     state: started
     enabled: true
 
+- name: Ensure Nginx is running
```

Comment:
Why is this needed? This restarts nginx every time the role is executed.

Reply:
Right, we can remove this task here. I added it since we needed to ensure that the nginx configurations are updated. However this should be part of the nginx role. I'll file a seperate PR for that and remove the task in here.